tests/umockcallrecorder_ut/umockcallrecorder_ut.c

```diff
@@ -553,6 +553,7 @@ TEST_FUNCTION(when_allocating_memory_fails_then_umockcallrecorder_create_fails)
 }
 
 /* Codes_SRS_UMOCKCALLRECORDER_01_097: [ If lock_factory_create_lock is not NULL, umockcallrecorder_create shall call lock_factory_create_lock to create the lock used when working with the stored calls. ]*/
```

Comment:
>Codes_SRS_UMOCKCALLRECORDER_01_097 [](http://example.com/codeflow?start=3&length=34)

This should not be Codes #Resolved

Reply:
Fixed - changed Codes_SRS_UMOCKCALLRECORDER_01_097 to Tests_SRS_UMOCKCALLRECORDER_01_097